utils/tools.py

```diff
@@ -205,3 +206,35 @@ def get_free_port():
         except OSError:
             port += 1
     raise IOError("no free ports")
+
+
+def retrieve_span_links(span):
```

Comment:
not sure if this is the best place to this method

Reply:
Right now, it's only used in `tests/test_distributed.py`, you can place it here as `_retrieve_span_links`. If it'll be used in another file, we'll think where we'llp ut it (very probably in LibraryInterface).